File: Examples/CSharp/ProgrammersGuide/Load-Save-Convert/ExportHTMLinStream.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;

namespace CSharp.ProgrammersGuide.Load_Save_Convert
{
    public class ExportHTMLinStream
    {
        public static void Run()
        {
            // ExStart:ExportHTMLinStream
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadSaveConvert();
            // load diagram
            Diagram diagram = new Diagram(dataDir + "ExportToHTML.vsd");
            // save resultant HTML directly to a stream
            MemoryStream stream = new MemoryStream();
            diagram.Save(stream, SaveFileFormat.HTML);
            // ExEnd:ExportHTMLinStream
        }
    }
}

File: Examples/VisualBasic/Load-Save-Convert/ExportHTMLinStream.vb

Imports Aspose.Diagram
Imports System.IO
Imports System

Public Class ExportHTMLinStream
    Public Shared Sub Run()
        ' ExStart:ExportHTMLinStream
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_LoadSaveConvert()
        ' Load diagram
        Dim diagram As New Diagram(dataDir & Convert.ToString("ExportToHTML.vsd"))
        ' Save resultant HTML directly to a stream
        Dim stream As New MemoryStream()
        diagram.Save(stream, SaveFileFormat.HTML)
        ' ExEnd:ExportHTMLinStream
    End Sub
End Class
